com.Acestream.engine.appdata.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.Acestream.engine</id>
```

Comment:
There's no such domain as `acestream.com`. Quick web search tells me about `acestream.media` and `acestream.org`.

Reply:
I apologize. This is my first project. org is the old domain. Now acestream.media is a new domain.